### PR TITLE
Release 3.0.0

### PR DESCRIPTION
**Pull request by TheMeier**
#### Pull Request (PR) description
Create release

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -4,24 +4,47 @@ All notable changes to this project will be documented in this file.
...
+[Full Changelog](https://github.com/voxpupuli/puppet-openvmtools/compare/v2.1.0...v3.0.0)
 
 **Breaking changes:**
 
-- Require puppet-epel over stahnma-epel [\#29](https://github.com/voxpupuli/puppet-openvmtools/pull/29) ([dhoppe](https://github.com/dhoppe))
+- Update from voxpupuli modulesync\_config; Drop RHEL 6; Drop Puppet 5 [\#53](https://github.com/voxpupuli/puppet-openvmtools/pull/53) ([TheMeier](https://github.com/TheMeier))
+
+**Closed issues:**
+
+- Add RockyLinux [\#50](https://github.com/voxpupuli/puppet-openvmtools/issues/50)
```
Comment (bastelfreak) [inline CHANGELOG.md:17]: I added the skip-changelog label here, please regenerate

File: CHANGELOG.md [hunk 2]
```diff
@@ -4,24 +4,47 @@ All notable changes to this project will be documented in this file.
...
+
+**Merged pull requests:**
+
+- feat: add Rocky Linux to supported OSes [\#51](https://github.com/voxpupuli/puppet-openvmtools/pull/51) ([TheMeier](https://github.com/TheMeier))
```
Comment (bastelfreak) [inline CHANGELOG.md:21]: I added enhancement here

File: metadata.json [hunk 3]
```diff
@@ -120,5 +120,5 @@
     "vmware-tools",
     "vmware_tools"
   ],
-  "version": "2.1.1-rc0"
+  "version": "3.0.1-rc0"
```
Comment (bastelfreak) [inline metadata.json:123]: please go to 3.0.0

File: CHANGELOG.md [hunk 4]
```diff
@@ -36,6 +59,18 @@ These should not affect the functionality of the module.
 - Fix Ubuntu [\#35](https://github.com/voxpupuli/puppet-openvmtools/pull/35) ([ghoneycutt](https://github.com/ghoneycutt))
 - modulesync 3.0.0 & puppet-lint updates [\#31](https://github.com/voxpupuli/puppet-openvmtools/pull/31) ([bastelfreak](https://github.com/bastelfreak))
 
+## [epel-dependency](https://github.com/voxpupuli/puppet-openvmtools/tree/epel-dependency) (2020-04-29)
```
Comment (bastelfreak) [inline CHANGELOG.md:62]: this is odd. why do we now see branches in the changelog. that shouldn't be the case :thinking: . let me delete them

Comment (bastelfreak) [reply CHANGELOG.md:62]: edit: oh that branch does not even exist in our repo

File: metadata.json (hunk 3 above)
Comment (TheMeier) [reply metadata.json]: This was added by the rake task as separate commit, after tagging it...
```
* 2b069c7 Christoph Maser 37 minutes ago  (tag: v3.0.0)  Release 3.0.0
* 2b069c7 Christoph Maser 37 minutes ago  (tag: v3.0.0)  Release 3.0.0
```
I just dropped the extra commit

File: CHANGELOG.md (hunk 1 above)
Comment (TheMeier) [reply CHANGELOG.md]: done

File: CHANGELOG.md (hunk 4 above)
Comment (TheMeier) [reply CHANGELOG.md:61]: It also does not exist in my fork or local checkout...

Comment (TheMeier) [reply CHANGELOG.md:61]: https://github.com/voxpupuli/puppet-openvmtools/tree/epel-dependency DOES exist

Comment (bastelfreak) [reply CHANGELOG.md:61]: vey odd, but I deleted it:
```
$ git push origin :epel-dependency
To github.com:voxpupuli/puppet-openvmtools.git
 - [deleted]         epel-dependency
```

Comment (TheMeier) [reply CHANGELOG.md]: Its gone but there is still the branch ubuntu-20-support 
https://github.com/voxpupuli/puppet-openvmtools/pull/54/files#diff-06572a96a58dc510037d5efa622f9bec8519bc1beab13c9f251e97e657a9d4edR41

**Review by bastelfreak — APPROVED** [review]

**Pull request merged**